packages/roosterjs-editor-types/lib/event/SelectionChangeEvent.ts

```diff
@@ -0,0 +1,50 @@
+import BasePluginEvent from './BasePluginEvent';
+import TableSelection from '../interface/TableSelection';
+import { PluginEventType } from '../enum/PluginEventType';
+import { SelectionRangeTypes } from '../enum/SelectionRangeTypes';
+import type { CompatibleSelectionRangeTypes } from '../compatibleEnum/SelectionRangeTypes';
+import type { CompatiblePluginEventType } from '../compatibleEnum/PluginEventType';
+
+/**
+ * Data of SelectionChangedEvent
+ */
+export interface SelectionChangedEventData {
```

Comment:
Actually I think we just need to pass selection type for performance. since in most case plugin doesn't really need the detailed selection info. If we really need, we can add later.

Reply:
Should we use the whole SelectionRangeEx or just pass the selection type?

Reply:
in ImageEdit we will consume the this plugin data, is better to call getSelectionRange to retrieve the selected image or store it in this event?